src/SASViyaApiClient.ts

```diff
@@ -1387,7 +1387,12 @@ export class SASViyaApiClient {
     return `/folders/folders/${folder.id}`
   }
 
-  private async getContextByName(
+  /**
+   * Returns a compute context.
+   * @param contextName - the name of the context to return.
```

Comment:
What is meant here by a compute context?  Perhaps we can provide an example?  It's confusing to me - are we returning the name (I guess not), the UUID, or the JSON representation of the compute context instance?